index.html

```diff
@@ -5,21 +5,170 @@
+                        <img class="icon" src="assets/lecture.svg" alt="">
+                    </div>
+                    <div class="card-info">
+                        <div class="primary-color title">
+                            Radhn Festival
+                        </div>
+                        <div class="content">General Radhn, the mightiest demigod of the shattering awaits you.
+                            Champions, prepare for battle. A celebration of war! THE RADHN FESTIVAL!!!</div>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="flex justify-center">
+                        <img class="icon" src="assets/lecture.svg" alt="">
+                    </div>
+                    <div class="card-info">
+                        <div class="primary-color title">
+                            Reward
+                        </div>
+                        <div class="content">Defeat the General. Claim glory and grab the great rune. The reward is THE
+                            STARSCOURAGE GREATSWORD!</div>
+                    </div>
+                </div>
+            </div>
+            <div class="underline">SEE THE WHOLE PROGRAM</div>
+        </div>
+    </section>
+    <section id="teachers" class="flex justify-center teachers">
+        <div class="bg">
+            <header class="flex justify-center">
+                <div>
+                    <div>
+                        Featured Teachers
+                    </div>
+                    <div class="flex justify-center">
+                        <div class="divider"></div>
+                    </div>
+                </div>
+            </header>
+            <div id="speakers-container" class="flex justify-center wrap">
+                <!-- js objects go here -->
+            </div>
+        </div>
+    </section>
+    <section id="partners" class="flex justify-center partners">
+        <div>
+            <div class="text-center" style="color: #999999; font-size: 30px; margin-top: 30px;">
+                Partners
+            </div>
+            <div class="flex justify-center">
+                <div class="divider"></div>
+            </div>
+            <div class="flex justify-center wrap">
+                <img src="assets/mozilla.png" alt="mozilla">
+                <img src="assets/google.png" alt="google">
+                <img src="assets/naver.png" alt="naver">
+                <img src="assets/daumkakao.png" alt="daumkakao">
+                <img src="assets/airbnb.png" alt="airbnb">
+            </div>
+        </div>
+    </section>
+    <footer class="flex justify-center">
+        <div class="flex items-center justify-center crd">
+            <h1>THE GOLDEN ORDER</h1>
+            <p>General Radhn, the mightiest demigod of the shattering awaits you.
+                Champions, prepare for battle. A celebration of war! THE RADHN FESTIVAL!!!</p>
+        </div>
+    </footer>
```

Comment:
- [ ] Kindly check on your design, there is a paragraph missing as shown below,:
<img width="542" alt="image" src="https://user-images.githubusercontent.com/61408860/174977044-24a68b08-eb3f-4442-923a-d09ef51aa13f.png">